lib/charms/observability_libs/v1/cert_handler.py

```diff
@@ -338,6 +349,11 @@ def ca_cert(self) -> Optional[str]:
         """CA Certificate."""
         return self._retrieve_from_secret("ca-cert", "secret-id")
 
+    @property
+    def ca_server_cert_secret_id(self) -> Optional[str]:
+        """CA, servert cert secret id."""
```

Comment:
Minor typo:

```suggestion
        """CA server cert secret id."""
```

Reply:
I'd make it more clear than just re-typing the function name:

```suggestion
        """ID of the Juju Secret for the CA server certificate."""
```